tests/test_integration_cross_signing_smoke.py

```
@@ -0,0 +1,65 @@
+        + """
+from nio.crypto.cross_signing import CrossSigningIdentity
+
+identity = CrossSigningIdentity.generate("@bot:example.org")
+payload = identity.self_signing_key_payload()
+signature = payload["signatures"][identity.user_id][
+    f"ed25519:{identity.master_public_key}"
+]
+verify_json(identity.master_public_key, payload, signature)
+"""
+    )
+    completed = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)